### PR TITLE
Fix squashing of webpage when width is 1100px-848px

### PR DESCRIPTION
**Pull request by moripen**
Adjusting breakpoint from 848px to 1100px so the padding looks natural on every resolution

### CODE REVIEW

**Comment by moripen** [comment]
Decided to also fix the favicon as we are currently using the default one.

File: public/bjerk-favicon.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<svg width="1119" height="1119" viewBox="0 0 1119 1119" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (cobraz) [inline public/bjerk-favicon.svg:1]: SVG favicon? Does that work?

Maybe we should run Prettier on this?

File: src/components/seo.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ export const SEO: React.FC<SEOProps> = ({
   return (
     <Head>
       <title>{seo.title}</title>
-      <link rel="icon" href="/favicon.ico" />
+      <link rel="icon" href="/bjerk-favicon.svg" />
```
Comment (cobraz) [inline src/components/seo.tsx:43]: Should probably be named favicon to adhere to standards

**Review by cobraz — APPROVED** [review]
Favicon changes are unrelated, right?

File: public/bjerk-favicon.svg (hunk 1 above)
Comment (moripen) [reply public/bjerk-favicon.svg:1]: We haven't ran Prettier on any of the SVG's, but yeah that might be an idea

File: src/components/seo.tsx (hunk 2 above)
Comment (moripen) [reply src/components/seo.tsx:43]: Favicon is just the name of the type of icon that is used in the tab, but as it is Bjerk's favicon - imo it's a fitting name.

**Comment by moripen** [comment]
The favicon changes are unrelated, but it has been on my mind.

File: src/components/seo.tsx (hunk 2 above)
Comment (cobraz) [reply src/components/seo.tsx:43]: If the metadata reference is not retrieved, the browser looks for a favicon (placed in the root directory, named "favicon")

Not sure if it works with SVG though.

Comment (moripen) [reply src/components/seo.tsx:43]: Ah, okay.

**Review by cobraz — APPROVED** [review]

**Review by cobraz — APPROVED** [review]

**Pull request merged**